src/Skia/Perspex.Skia.Android/AndroidRenderTarget.cs

```diff
@@ -14,6 +14,8 @@
 
 namespace Perspex.Skia
 {
+/* This lives in shared project now, but we may need an android specific implementation anyways
```

Comment:
I've removed the comments, but kept the class (partial) as we will likely need it for the HW accel.